src/main/java/edu/kis/powp/jobs2d/drivers/adapter/BasicLineDrawOperator.java

```diff
@@ -5,13 +5,15 @@
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
 
+import static edu.kis.powp.jobs2d.features.DrawerFeature.getDrawerController;
+
 /**
- * driver adapter to drawer with several bugs.
+ * basic driver operator to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class BasicLineDrawOperator extends DrawPanelController implements Job2dDriver {
```

Comment:
Why Operator not Driver?
Inheritance is not used (no super drawLine calls). Should be removed!